packages/ui/src/index.tsx

```diff
@@ -59,7 +59,40 @@ export * from './components/TamaguiProvider';
 export * from './components/UrbitSigil';
 export * from './components/View';
 export * from './components/WelcomeSheet';
+export * from './components/Modal';
 export * from './contexts';
 export * from './tamagui.config';
 export * from './types';
 export * from './utils';
+
+export {
+  Circle,
+  useTheme,
+  useStyle,
+  View,
```

Comment:
When should we do `import { View } from 'ui'` instead of `import { View } from 'tamagui'`?